src/utils/user_utils.ts

```diff
@@ -160,8 +160,7 @@ export function filterProfilesMatchingTerm(users: Array<UserProfile>, term: stri
         }
 
         return profileSuggestions.
-            filter((suggestion) => suggestion !== '').
-            some((suggestion) => suggestion.startsWith(trimmedTerm));
+            filter((suggestion) => suggestion !== '')      .some((suggestion) => suggestion.indexOf(trimmedTerm) >= 0);
```

Comment:
Can we move this to new line?
```suggestion
            filter((suggestion) => suggestion !== '').
            some((suggestion) => suggestion.indexOf(trimmedTerm) >= 0);
```

Reply:
Hi,

Thanks. Shouldn't be problem.  Thanks for looking into it.